lib/node_modules/@stdlib/blas/base/dsyr/README.md

```diff
@@ -47,11 +47,11 @@ dsyr( 'row-major', 'upper', 3, 1.0, x, 1, A, 3 );
 The function has the following parameters:
 
 -   **order**: storage layout.
--   **uplo**: specifies whether the upper or lower triangular part of the symmetric matrix `A` should be referenced.
+-   **uplo**: specifies whether the upper or lower triangular part of the symmetric matrix `A` is supplied.
```

Comment:
This change should be reverted. "should be referenced" is intentional and follows the BLAS docs.